snippets/plugin-base-template.py

```
@@ -10,8 +10,8 @@
     4. Implement your plugin logic in the appropriate methods
 
 Requirements:
-    - Indigo 2023.2+ (Python 3.10+)
-    - ServerApiVersion 3.0 in Info.plist
+    - Indigo 2023.2+ (Python 3.10–3.13; 2025.2 recommended)
```

Ruff (0.15.12)

[warning] Docstring contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF002)